n3fit/runcards/reproduce_nnpdf40_updated/nnpdf40_updated.yml

```diff
@@ -0,0 +1,264 @@
+name: nnpdf40_updated
+channels:
+  - https://packages.nnpdf.science/public
+  - conda-forge
+  - defaults
```

Comment:
Shouldn't this be `defaults` then `conda-forge`? as in https://github.com/NNPDF/binary-bootstrap

Reply:
Is there a good reason `defaults` is first in binary bootstrap? I think it doesn't matter much

Reply:
That it is what god intended.

More seriously, don't know. But I guess the `reproduce_xxx` and the binary bootstrap should align. The fact that you are running with this and it works (also the qed fits are running with this environment I guess?) means that probably updating the bootstrap repository is quicker.

Reply:
I guess this is just the way it was ordered in my `.condarc` when I produced this file. In the env for nnpdf4.0 it's actually the other way around. 

I guess (it's really not much more then a guess) that when creating the env this overrides the `.condarc` (or at least the `channels`). There will always remain some ambiguity e.g. the defaults for strict channel priority I believe can depend on the specific version of conda that's being used. 

That being said, I assume the build numbers differ between the different channels and thus there is a unique solution to this configuration of packages that does not depend on the order of the channels. Maybe that's not the case?